kb-importer/src/main/java/org/eclipse/steady/kb/command/CommandExecutor.java

```diff
@@ -120,18 +120,9 @@ private void printHelp(Options commandOptions) {
    * get directory absolute path if it is relative
    *
    * @param rootDir
-   * @return
+   * @return absolute path
    */
   private String getAbsolutePath(String rootDir) {
-    Path p = Paths.get(rootDir);
```

Comment:
This can be simplified by using [`Path.resolve(Path)`](https://docs.oracle.com/javase/7/docs/api/java/nio/file/Path.html#resolve(java.nio.file.Path))

Reply:
Hi Henrik,
But we do not want to resolve against any path. We just want the absolute path from where our java program is run. And I think toAbsolutePath does well here.

Reply:
Example - 
if I run the java command -
`~/Downloads/kaybee> java -jar /Users/test/Documents/kb-importer/target/kb-importer-3.1.14-SNAPSHOT-jar-with-dependencies.jar -d ./CVE-2018-1000519-test`

The -d option path should be resolved as `/Users/test/Downloads/kaybee/CVE-2018-1000519-test`